fcomm/src/bin/fcomm.rs

```diff
@@ -520,9 +522,14 @@ where
 fn main() {
     let cli = Cli::parse();
 
-    pretty_env_logger::formatted_builder()
-        .filter_level(cli.verbose.log_level_filter())
-        .init();
+    let subscriber = Registry::default()
+        // TODO: correctly filter log level with `clap_verbosity_flag`
+        .with(fmt::layer().pretty())
+        .with(EnvFilter::from_default_env())
+        // note: we don't `tracing_texray::examine` anything below, so no spans are printed
+        // but we add the layer to allow the option in the future, maybe with a feature?
+        .with(TeXRayLayer::new());
```

Comment:
Same thing, I'd be shocked if we used fcomm to drive our micro-benchmarking, so perhaps TexRay is superfluous? the rest above is good.